app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -101,6 +101,13 @@ define([
         show: function (record) {
             var img;
 
+            if (record._rowIndex === this.visibleRecord() &&
+                this.isVisible(record)) {
```

Comment:
There is no need to call `this.isVisible(record)` as the first condition seems to be enough

Reply:
ok